content/apps/update-scopes.md

```diff
@@ -0,0 +1,36 @@
+# Update access scopes
+
+::: warning
+**This feature is only available on SaaS platforms.**
+:::
+
+It's possible to update the access scopes of an already connected app.
+The access scope update process requires app users to consent to new scopes changes. 
+
+If your app needs to change the initial scope list, it must initiate an authorization request with the entire list of access scopes your app needs. The authorization request follows the usual OAuth 2.0 protocol through which the app will be able to retrieve a new Access Token that reflects new requested scopes.
+
+Through Akeneo REST API, connected apps can notify Akeneo PIM users who can manage your app that the authorization scopes your app requires have changed. 
+
+To do so, use the following `POST` method, with parameters below:
+
+```
+https://my-pim.cloud.akeneo.com/connect/apps/v1/scopes/update?scope=[REQUESTED_SCOPES]
+```
+Requirements:
+- `scopes`: query parameter to specify app new scope list as a space-separated string
+- `Authorization: Bearer [AccessToken]`: header to authenticate your app
+- `Content-Type: application/json`: header for the response format
+
+Response: `"Ok"`
+
+The notification will warn them and entice them to open your app so that you can initiate an authorization request with the entire list of required scopes.
```

Comment:
maybe display a screenshot of the notification ?

Reply:
yes now i have a screenshot ;)